eShopModernizedWebFormsSolution/src/eShopModernizedWebForms/Web.config

```diff
@@ -5,21 +5,31 @@
   -->
 <configuration>
   <configSections>
+    <!-- For more information on Entity Framework configuration, visit http://go.microsoft.com/fwlink/?LinkID=237468 -->
     <section name="entityFramework" type="System.Data.Entity.Internal.ConfigFile.EntityFrameworkSection, EntityFramework, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" requirePermission="false" />
-    
-  <!-- For more information on Entity Framework configuration, visit http://go.microsoft.com/fwlink/?LinkID=237468 --></configSections>
-  <connectionStrings>
+    <section name="configBuilders" type="System.Configuration.ConfigurationBuildersSection, System.Configuration, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" restartOnExternalChanges="false" requirePermission="false" />
+  </configSections>
+  <configBuilders>
+    <builders>
+      <add name="Environment" type="Microsoft.Configuration.ConfigurationBuilders.EnvironmentConfigBuilder, Microsoft.Configuration.ConfigurationBuilders.Environment, Version=2.0.0.0, Culture=neutral" />
+      <add name="AzureKeyVault" vaultName="${KeyVaultName}" optional="true" type="eShopModernizedWebForms.OptionalKeyVaultConfigurationBuilder, eShopModernizedWebForms" />
+      <add name="Secrets" userSecretsId="5702632e-73c6-478a-a875-2714c09d921b" type="Microsoft.Configuration.ConfigurationBuilders.UserSecretsConfigBuilder, Microsoft.Configuration.ConfigurationBuilders.UserSecrets, Version=2.0.0.0, Culture=neutral" />
```

Comment:
We should sanitize this userSecretsId so that it isn't checked in.

Reply:
userSecretsId isn't a real secret - it's an identifier for a folder on disk where secrets could be saved.